client/src/redux/actions/scansActions.js

```diff
@@ -1,61 +1,77 @@
 import { scansTypes } from '../constants';
 import scansService from '../../services/scansService';
 
+const dispatchObjects = {
+  addScan: data => {
+    return {
```

Comment:
Note: see if we can figure out a easier way of handling the tests... this abstraction feels a little wonky, hard to read